packages/core/src/testing/assertions.rs

```diff
@@ -59,6 +59,9 @@ pub fn assert_approx_eq_impl<U: Into<Uint128>>(
     let max_rel_diff = Decimal::from_str(max_rel_diff).unwrap();
 
     let largest = core::cmp::max(left, right);
+    if largest.is_zero() {
+        return;
+    }
```

Comment:
`largest` is zero if and only if both `left` and `right` are zero (since both are Uint128 at this point). I think checking

```rust
if left.is_zero() && right.is_zero() {
    return; // values equal, all good, exit early
}
```

or even

```rust
if left == right {
    return; // values equal, all good, exit early
}
```

would achieve the same but is easier to review

Reply:
True, easier to read and reason about. Can update it.